### ISSUE TITLE
Incorrect role is defined for menuitems "User", "Workspace" and "Remote[Azure ML: v-...]" as checkboxes.: A11y_VS Code Client_Settings_AI4D

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#45578]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

--- GitHub Tags
-A11yTCS; -A11ySev2; -WCAG4.1.2; -BM-VisualStudioCodeClient-Win32-Jan2024; -DesktopApp; -Visual Studio Code Client; -A11yMAS; -Name Role Value; -AILimited; -Win11; -AI4D;

--- Environment Details:
Application Name: Visual Studio Code Client
Visual Studio Code: Version 1.89.1(User setup)

--- Repro Steps:

1. Install VS Code
2. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
3. Open the Command Palette via the keyboard command Ctrl+Shift+P (or Command+Shift+P for MacOS)
4. Type "Open User Settings" and select the option from the dropdown list called "Preferences: Open User Settings") and press enter
5. In the search box, type "Azure Machine Learning"
6. Tab until the list of settings is active
7. Use the up/down arrows to navigate between settings
8. Tab till "User" menuitem and Verify whether the correct role is defined for "User", "Workspace" and "Remote[Azure ML: v-...]" or not.

--- Actual Result:
Incorrect role is defined for menuitems "User", "Workspace" and "Remote[Azure ML: v-...]" as checkboxes

--- Expected Result:
Correct role should be defined for "User", "Workspace" and "Remote[Azure ML: v-...]" as menuitems.

--- User Impact:
Screen reader users will face difficulty in accessing the control if the correct control type is not defined for the controls.

--- Attachment:
![image](https://github-com/microsoft/vscode/assets/98530069/477a6ca7-e241-4e13-a600-2a69cad227e2)
